packages/toast/src/components/ToastComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    EDSStyleSheet,
+    Icon,
+    Paper,
+    Typography,
+    useStyles,
+    useToken,
+} from "@equinor/mad-components";
+import React from "react";
+import { TouchableHighlight } from "react-native";
+import { ToastType } from "../types";
+import { getMaxToastWidth } from "../utils/getMaxToastWidth";
+import { getToastTokenForType } from "../utils/getToastTokenForType";
+
+export type ToastComponentProps = { text: string; type: ToastType; onPress?: () => void };
+export const ToastComponent = ({ text, type, onPress }: ToastComponentProps) => {
+    const styles = useStyles(theme, type);
+    const masterToken = useToken();
+    const { text: textColor } = getToastTokenForType(masterToken, type);
+
+    return (
+        <TouchableHighlight
+            disabled={typeof onPress !== "function"}
+            onPress={onPress}
+            style={{ borderRadius: 4 }}
+        >
+            <Paper elevation="raised" style={styles.background}>
+                <Icon name="information-outline" color={textColor} />
```

Comment:
Should the icon either be configurable, or at least change based on the type of toast?
the information-outline for error, warning and success is maybe not so fitting

Reply:
@EmilieOlsen What do you think?

Reply:
So sorry for the late reply! Didn't see this. 
I didn't think much of it until now and it makes sense. EDS has a separate error and a warning icon, so it makes sense to change it. 

<img width="432" alt="image" src="https://github.com/equinor/mad/assets/111871753/47bed943-8288-490b-a211-224ecdf939e2">